modules/alloy/src/main/ts/ephox/alloy/behaviour/blocking/BlockingSchema.ts

```diff
@@ -0,0 +1,31 @@
+import { FieldSchema } from '@ephox/boulder';
+import { Optional } from '@ephox/katamari';
+import * as Fields from '../../data/Fields';
+import * as Behaviour from '../../api/behaviour/Behaviour';
+import { AlloyComponent } from '../../api/component/ComponentApi';
+
+export interface BlockingConfigSpec extends Behaviour.BehaviourConfigSpec {
```

Comment:
Readonly and move this to a types file please

Reply:
Well I made it readonly?

Reply:
Don't git and drive, I guess